examples/fluent-bit/multi-config-support/task-definition.json

```diff
@@ -0,0 +1,77 @@
+{
+	"family": "firelens-example-multi-config-support",
+	"taskRoleArn": "arn:aws:iam::xxxxxxxxxx:role/ecs_task_iam_role",
+	"executionRoleArn": "arn:aws:iam::xxxxxxxxxx:role/ecs_task_execution_role",
+	"containerDefinitions": [
+		{
+			"essential": true,
+			"image": "public.ecr.aws/aws-observability/aws-for-fluent-bit:init-latest",
+			"name": "log_router",
+			"firelensConfiguration": {
+				"type": "fluentbit"
+			},
+            "mountPoints" : [
+				{
+					"containerPath" : "/logs/",
+					"sourceVolume" : "log"
+				}
+			],
+			"logConfiguration": {
+				"logDriver": "awslogs",
+				"options": {
+					"awslogs-group": "firelens-container",
+					"awslogs-region": "us-west-2",
+					"awslogs-create-group": "true",
+					"awslogs-stream-prefix": "firelens-init"
+				}
+			},
+            "environment": [
```

Comment:
the JSON formatting is a bit messed up, run this through something like this to make it pretty: https://jsonformatter.org/json-pretty-print

Reply:
That's a great tool, the format now pretty standard!